prac_05/word_occurrences.py

```diff
@@ -0,0 +1,17 @@
+"""
+Word Occurrences
+Estimate: 10 minutes
+Actual: 22 minutes
+"""
+word_to_frequency = {}
+# text = input("Enter Text: ")
+text = "this is a collection of words of nice words this is a fun thing it is"
+words = text.split()
+for word in words:
+    frequency = word_to_frequency.get(word, 0)
+    word_to_frequency[word] = frequency + 1
+words = list(word_to_frequency.keys())
+words.sort()
+longest_word = max((len(word) for word in words))
```

Comment:
As you're finding the maximum length, is the name longest_word misleading?

Reply:
Good observation